eoflow/models/conv_cells.py

```diff
@@ -1,6 +1,6 @@
 """ Implementation of ConvLSTM and ConvGRU derived from https://github.com/carlthome/tensorflow-convlstm-cell """
```

Comment:
This entire file is also deprecated and should be removed. If we update it we can include it later.